src/overlays/actors/ovl_Bg_Ikana_Dharma/z_bg_ikana_dharma.c

```diff
@@ -28,47 +34,226 @@ const ActorInit Bg_Ikana_Dharma_InitVars = {
     (ActorFunc)BgIkanaDharma_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BECAD0 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_NONE, OC2_NONE, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x00000100, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0x00000100, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_NONE,
+    },
     { 98, 10, 25, { 0, 0, 0 } },
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80BECAFC[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneForward, 4000, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneScale, 320, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneDownward, 320, ICHAIN_CONTINUE),
     ICHAIN_F32_DIV1000(gravity, -1100, ICHAIN_STOP),
 };
 
-#endif
+static BgIkanaDharma* sFirstHitBgIkanaDharma;
 
-extern ColliderCylinderInit D_80BECAD0;
-extern InitChainEntry D_80BECAFC[];
+void BgIkanaDharma_CreateParticles(BgIkanaDharma* this, GlobalContext* globalCtx) {
+    s32 pad;
+    Vec3f pos;
+    Vec3f velocity;
+    Vec3f accel;
+    s32 i;
+    f32 initialRadius = this->dyna.actor.scale.x * 200.0f;
+    f32 initialY = this->dyna.actor.scale.y * 50.0f;
 
-extern UNK_TYPE D_060008C8;
-extern UNK_TYPE D_06000C50;
+    for (i = 0; i != 4; i++) {
```

Comment:
```suggestion
    for (i = 0; i < 3; i++) {
```
That doesnt match does it?

Reply:
Close, `i < 4` matches.